sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -647,7 +651,7 @@ private case class GpuParquetFileFilterHandler(@transient sqlConf: SQLConf) exte
       val fileSchema = footer.getFileMetaData.getSchema
       val pushedFilters = if (enableParquetFilterPushDown) {
         val parquetFilters = SparkShimImpl.getParquetFilters(fileSchema, pushDownDate,
```

Comment:
We're already shimming `getParquetFilters`, would it make sense to pass the conf and have the shim pull out the appropriate string predicate conf rather than have this code need to call a shim just to get a parameter to call another shim?

Reply:
We do this to avoid serializing the SQL conf object to executors, and send only the necessary confs to executors.
The config is pulled out at driver side, while the `getParquetFilters` runs on executors.